tools/compare-wp-performance/old/package.json

```diff
@@ -0,0 +1,24 @@
+{
+	"name": "@compare-wp-performance/old",
+	"private": true,
+	"description": "Comparing performance between two WordPress core releases with benchmarking",
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/swissspidy/compare-wp-performance"
+	},
+	"author": "swissspidy",
+	"license": "Apache-2.0",
+	"bugs": {
+		"url": "https://github.com/swissspidy/compare-wp-performance/issues"
+	},
+	"homepage": "https://github.com/swissspidy/compare-wp-performance#readme",
+	"type": "module",
+	"devDependencies": {
+		"@wordpress/env": "^8.4.0",
+		"patch-package": "^7.0.0"
+	},
+	"scripts": {
+		"postinstall": "patch-package",
+		"wp-env": "wp-env"
+	}
+}
```

Comment:
Can you clarify why we need the `package.json` twice, one for the `new` and one for the `old` folder? Couldn't those share the dependencies between a single `package.json`?

Reply:
A locally installed wp-env only works in the current folder as that‘s where it‘s looking for the config file